jepsen/src/jepsen/control/util.clj

```diff
@@ -30,10 +30,13 @@
             opts))))
 
 (defn file?
+  "Is `filename` a regular file that exists?"
   [filename]
-  (throw (RuntimeException. "Use exists? instead; file? will be used to tell if
-                            something is a file, as opposed to a directory or
-                            link.")))
+  (try+
+   (exec :test :-f filename)
+   true
+   (catch [:exit 1] _
+     false)))
```

Comment:
Yup, I'm all for this. Been more than long enough. :-)

Reply:
Oh good, as I've been using it in other places too. :-)